karyon3-admin-simple/src/main/java/com/netflix/karyon/admin/StaticResource.java

```diff
@@ -0,0 +1,31 @@
+package com.netflix.karyon.admin;
+
+/**
+ * Encapsulate a resource loaded in {@link StaticResourceProvider} 
+ * 
+ * @author elandau
+ */
+public class StaticResource {
+    private final String mimeType;
+    private final byte[] data;
+
+    public static final StaticResource INVALID = new StaticResource(null, null);
```

Comment:
From above, I'd rather return an Option<StaticResource> than have this INVALID static.